assets/js/workflow-diagram/WorkflowDiagram.tsx

```diff
@@ -44,10 +41,12 @@ type ChartCache = {
 
 const LAYOUT_DURATION = 300;
 
-export default (props: WorkflowDiagramProps) => {
-  const { selection, onSelectionChange, store, el } = props;
+export default function WorkflowDiagram(props: WorkflowDiagramProps) {
+  const { jobs, triggers, edges, disabled } = useWorkflowStore();
+  const { selection, onSelectionChange, containerEl: el } = props;
```

Comment:
What if `usePlaceholder` attaches its listeners to `document.body`? That way we should be able to remove this element reference entirely? Which feels way cleaner

The downside is you can't have two editors in one page. But I think the way this is set up right now you can't really do that anyway?

You could namespace the event with an id if you needed two stores - that would work